src/main/java/no/ndla/taxonomy/rest/v1/commands/NodePostPut.java

```diff
@@ -58,6 +62,12 @@ public class NodePostPut implements UpdatableDto<Node> {
     @Schema(description = "The language used at create time. Used to set default translation.", example = "nb")
     public Optional<String> language = Optional.empty();
 
+    @JsonProperty
+    @Schema(description = "The quality evaluation of the node. Consist of a score from 1 to 5 and a comment.")
+    @JsonDeserialize(using = QualityEvaluationDTODeserializer.class)
+    @NullOrUndefined
```

Comment:
Denne tilsvarer UpdateOrDelete i backend?

Reply:
Den er ikke like smart som i backenden, men det er for å få til samme funksjonalitet.

Akkurat den `@NullOrUndefined` er bare for å gi typen mulighet til å både være `null` og `undefined`, men den custom deserializeren definert med `@JsonDeserialize` er for å faktisk ha muligheten til å ha `null`, `Optional.empty` og `Optional.of`.

Reply:
Kan ikkje dette heller løses ved å sende inn 0 i qualityEvaluation grade? Grade er jo definert som en verdi mellom 1 og 5. Då slipper vi spesialhåndteringa.

Reply:
Hmm, jeg elsker ikke det apiet heller.
La meg teste og se om jeg klarer å lage en som minner mer om `UpdateOrDelete` i fra backend.

Reply:
Viser seg at custom deserializers ikke kalles når verdien mangler.
Så jeg tror vi blir nødt til å ha noe nullability her. Kan droppe `Optional` og heller lage en custom klasse da.

Reply:
Dette ser ut til å være bedre måte å håndtere det på ja.